models/agol.js

```diff
@@ -254,6 +266,46 @@ var AGOL = function (koop) {
   }
 
   /**
+   * Wraps Cache.updateInfo for easier testing
+   * @param {string} key - a table name in the db
+   * @param {object} info - the info to store in the table
+   * @param {function} callback  - calls back with err or nothing
+   */
+  agol.updateInfo = function (key, info, callback) {
+    koop.Cache.getInfo(key, info, callback)
```

Comment:
@dmfenton - I'm waaaay off the back on koop. But I'm not sure this is right. The comment says "Wraps Cache.updateInfo..." and the method is called `updateInfo` but you actually wrap `getInfo`.